miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -767,6 +810,26 @@ export.get_note_serial_number
     # => [SERIAL_NUMBER]
 end
 
+#! Tells the transaction kernel that we are about to execute a procedure on a foreign account.
+#!
+#! Panics if we are already in a foreign context.
+#!
+#! Inputs:  [foreign_account_id, pad(15)]
+#! Outputs: [pad(16)]
+export.start_foreign_context
+    drop # unimplemented!
```

Comment:
This is left for a future PR, right?

Reply:
Yes, implementation of this function highly depends on how we will provide the account data, so I decide to move it to the next PR.